frontend/tests/integration/specs/work-packages/details-pane/details-pane-helper.js

```diff
@@ -38,7 +39,92 @@ function showAll() {
   $('.panel-toggler a').click();
 }
 
+function behaveLikeEmbeddedDropdown(name, correctValue) {
+  context('behaviour', function() {
+    var editor = $('.inplace-edit.attribute-' + name);
+
+    before(function() {
+      loadPane(819, 'overview');
+      showAll();
+    });
+
+    describe('read state', function() {
+      it('should render a span with value', function() {
+        expect(
+          editor
+          .$('.inplace-edit--read-value')
+          .getText()
+        ).to.eventually.equal(correctValue);
+      });
+    });
+
+    describe('edit state', function() {
+      before(function() {
+        editor.$('.inplace-editing--trigger-link').click();
+      });
+
+      context('dropdown', function() {
+        it('should be rendered', function() {
+          expect(
+            editor
+            .$('.select2-container').isDisplayed()
+            .isDisplayed()
```

Comment:
I removed it as well and the test still runs fine - i think it's a copy pasta error.

Log story short - remove it, after that this is good to merge :)